projects/components/src/datagrid/datagrid.component.html

```diff
@@ -111,22 +111,24 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
             </div>
         </clr-dg-cell>
 
-        <clr-dg-cell *ngFor="let column of columnsConfig">
-            <!-- Default renderer -->
-            <ng-container *ngIf="column.fieldName">{{ restItem | nestedProperty: column.fieldName }}</ng-container>
+        <div *ngFor="let column of columnsConfig">
```

Comment:
Why the wrapping div? If really needed, it should be an `ng-container` to avoid adding an HTML element but I thought you should be able to use it in the same tag as the *ngFor

```
<clr-dg-cell *ngFor="let column of columnsConfig; let column = column" [vcdShowClippedText]="column.cliptextConfig">

```